src/modules/flow/iio/nodes.c

```diff
@@ -143,6 +143,11 @@ iio_direction_vector_reader_cb(void *data, struct sol_iio_device *device)
     r = sol_iio_read_channel_value(mdata->channel_z, &out.z);
     SOL_INT_CHECK_GOTO(r, < 0, error);
 
+    SOL_DBG("Before mount_caliration: %f-%f-%f", out.x, out.y, out.z);
+
+    // mount correction
+    sol_iio_mount_calibration(device, &out.x, &out.y, &out.z);
```

Comment:
@edersondisouza Based on my understanding, the mount_matrix calibraton needs all these X/Y/Z raw data, only nodes.c can capture all these X/Y/Z data together.Is there any a proper location internally for the X/Y/Z data calibration?